applications/GeoMechanicsApplication/tests/test_reset_displacement.py

```diff
@@ -37,7 +37,7 @@ def test_reset_displacement_truss(self):
         eps = F/(E*A)
 
         # get stages
-        test_name = 'geo_truss_with_reset_displacemnet'
+        test_name = 'geo_truss_with_reset_displacement'
         project_path = test_helper.get_file_path(os.path.join('.', test_name + '.gid'))
```

Comment:
I want to suggest removing these ".gid" extensions when we revisit tests, as they were for the times that all the related gid files were also committed. But we can discuss it if you think it's extra work.

Reply:
Removed